src/main/java/io/confluent/connect/jdbc/source/TableQuerier.java

```diff
@@ -107,12 +109,24 @@ public boolean next() throws SQLException {
   public void reset(long now) {
     closeResultSetQuietly();
     closeStatementQuietly();
+    releaseLocksQuietly();
```

Comment:
should this be in a finally block, in case any of the above fail?

Reply:
All of the other `Quietly` functions here swallow the errors, so they won't fail in this block.